### ISSUE TITLE
Google SSO

### ISSUE THREAD

**Issue by vnbx** [sampsyo/hass-smartthinq#53]
when running the python3 example.py script, I go to the http location provided, and click google signin (google SSO) at that point I just get a Page not found message from the LG site.

**Comment by sampsyo** [comment]
Indeed, we've heard similar reports—I don't think this works, and I'm not sure why. Any help would be appreciated!

**Comment by vnbx** [comment]
I guess I will try to create a regular non-sso account in the meantime. thanks!

**Comment by malwcal** [comment]
I have the same problem. Is there a way to move an account over to using a password? Or do I need to re do everything again to set up my appliances in thinq.
